src/ngraph/op/fused/gelu.hpp

```diff
@@ -43,6 +43,9 @@ namespace ngraph
 
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
+
+        protected:
+            void generate_adjoints(autodiff::Adjoints& adjoints, const NodeVector& deltas) override;
```

Comment:
In the parent class, I see this. Is it ok to override a 'throw' ing unimplemented function? :  
```
void op::util::FusedOp::generate_adjoints(autodiff::Adjoints& adjoints, const NodeVector& deltas)
{
    // TODO
    throw ngraph_error("Autodiff on fused ops not supported yet");
}
```

Reply:
Yes, particularly since it is implementing it.

Reply:
Then maybe the throw comment is misleading?